Documentation/contributing.rst

```diff
@@ -213,6 +213,17 @@ commands, respectively:
 Testsuite
 ~~~~~~~~~
 
+.. Warning::
+
+  Running the testsuite will modify the host environment. If you are using the
+  default VM that might not be an issue, but if you are running bare metal or a
+  different VM the tests might fail or worst case remove possibly important
+  configuration. Specifically, they are modifying tables and flushing chains,
```

Comment:
Are these IPTables tables / chains?

Reply:
Yes

Reply:
OK, it might be good to name-drop iptables in this paragraph.

Reply:
fixed